src/dbg/commandline.cpp

```diff
@@ -103,7 +103,7 @@ void CmdLineCacheLoad(JSON Root)
     EXCLUSIVE_ACQUIRE(LockCmdLine);
 
     // Clear command line
-    memset(commandLine, 0, MAX_COMMAND_LINE_SIZE);

```

Comment:
the `commandLine` is declared as `char commandLine[MAX_SETTING_SIZE];` so this is correct.